pokecli.py

```diff
@@ -208,7 +209,13 @@ def main():
         logger.log('[x] Starting PokemonGo Bot....', 'green')
 
         while True:
-            bot.take_step()
+			try:
+				bot.take_step()
+			except NotLoggedInException:
+				logger.log('pgoapi is reporting that you are no longer logged in. Reinitializing...', 'red')
+				# re-initialize the bot wit the same config.
+				bot = PokemonGoBot(config)
```

Comment:
I'm not intimately familiar with the bot's internals yet, this was the
easiest way I could think of.
Currently it crashes and I manually run it again.. I figure this does the
same thing, just automates it.
The garbage collector should take care of the old PokemonGoBot object.

On Wed, Jul 27, 2016, 5:51 PM Douglas Camata <notifications@github.com>
wrote:

> In pokecli.py
> <https://github.com/PokemonGoF/PokemonGo-Bot/pull/1268#discussion_r72538750>
> :
>
> > @@ -208,7 +209,13 @@ def main():
> >          logger.log('[x] Starting PokemonGo Bot....', 'green')
> >
> >          while True:
> > -            bot.take_step()
> > +			try:
> > +				bot.take_step()
> > +			except NotLoggedInException:
> > +				logger.log('pgoapi is reporting that you are no longer logged in. Reinitializing...', 'red')
> > +				# re-initialize the bot wit the same config.
> > +				bot = PokemonGoBot(config)
>
> Why creating another bot instance? Just to reload config?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PokemonGoF/PokemonGo-Bot/pull/1268/files/9daacea1299160ed672ee14e56994051463c8eb8#r72538750>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD9PCjUKG0GppYOv59P6A4QxDKrYPmqYks5qZ-DtgaJpZM4JWp_j>
> .
>